edid-parse.c

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
This file should be in same coding style as the rest of the xplugd project.  This is less important than the other points (below), but I'm mentioning it anyway if you get the time to call [indent](https://github.com/troglobit/toolbox/blob/master/indent.sh) on it.